app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: "Teach in England if you trained outside the UK"
+subcategory: Teach in England as a non-UK citizen
```

Comment:
@laurahermionetennant92 Could this be something like If you're already a teacher? to make it different to the title on the card?